app/models/product.rb

```diff
@@ -12,9 +12,10 @@ class Product < ApplicationRecord
   validates :price, presence: true, numericality: { greater_than_or_equal_to: 1 }
   validates :link, presence: true
   validates :email, presence: true
+  validates :description, presence: true
   validates :novelty,
-    numericality: { greater_than: 0, less_than_or_equal_to: 5 },
-    allow_blank: true
+            numericality: { greater_than: 0, less_than_or_equal_to: 5 },
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.